src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java

```diff
@@ -313,5 +313,16 @@ public void inserimentoCodiceDelegaACaricoDellImpresaAPI(String codiceDelega){
 
     }
 
+    public boolean siVisualizzaUnaDelegaPG() {
+        try {
+            By menuDelega = By.xpath("//tr[contains(@class,'MuiTableRow-root css-g76qb5')]");
```

Comment:
If `id` or `data-testid` are available, use those instead of CSS selectors. CSS class names might change frequently during updates to dependencies or when modifying components.

Check if the element already exist otherwise just use the `data-testid=delegationMenuIcon`

Reply:
```suggestion
            By menuDelega = By.xpath("//button[@data-testid="delegationMenuIcon"]");
```